sdk/storage/src/common/storage_error.cpp

```diff
@@ -15,16 +16,8 @@ namespace Azure { namespace Storage {
       Azure::Core::Context context,
       std::unique_ptr<Azure::Core::Http::RawResponse> response)
   {
-    std::vector<uint8_t> bodyBuffer;
-    auto bodyStream = response->GetBodyStream();
-    if (bodyStream)
-    {
-      bodyBuffer = Azure::Core::Http::BodyStream::ReadToEnd(context, *bodyStream);
-    }
-    else
-    {
-      bodyBuffer = std::move(response->GetBody());
-    }
+    unused(context);
```

Comment:
didn't we decide not to use fn(..) to set an un-used var?
I remember we were going to either use `void(variable)` or `MACRO(variable)`

No need to fix for this PR. Let's follow up with @CaseyCarter @BillyONeal @JeffreyRichter @RickWinter